src/util/util.go

```diff
@@ -13,3 +13,22 @@ func GetCoinValue(value string) (uint64, error) {
 	coin, err := strconv.ParseFloat(value, 64)
 	return uint64(coin * 1000000), err
 }
+
+func Format(n uint64) string {
```

Comment:
`Format(n)` => `FormatCoins(coins, divisor)` . In code assume second argument will always be `1000000` for `SKY` ticker and `1` for `SKYCH`.